data/metadataSchema.json

```diff
@@ -0,0 +1,163 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "title": "Field Metadata for CSAF Documents",
+  "description": "JSON schema for json file storing metadata on fields of CSAF documents",
+  "type": "object",
+  "$defs": {
+    "relevance_levels_t": {
+      "title": "Relevance Levels",
+      "description": "Defines the possible relevance levels",
+      "type": "string",
+      "enum": [
+        "mandatory",
+        "best_practice",
+        "want_to_have",
+        "nice_to_know",
+        "optional",
+        "excluded"
+      ]
+    },
+    "documentation_file_path_t": {
+      "title": "File Path",
+      "description": "Defines the path to a file on the file system",
+      "type": "string",
+      "pattern": "docs/user(/.+)+-(spec|usage)(-csaf_(base|security_incident_response|informational_advisory|security_advisory|vex))?\\.[a-z]{2}\\.md"
+    }
+  },
+  "properties": {
+    "metaData": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "required": [
+          "path",
+          "user_documentation",
+          "description",
+          "label"
+        ],
+        "properties": {
+          "path": {
+            "title": "Json Path",
+            "description": "The json path of the field this item holds metadata for",
```

Comment:
Please always write JSON in capitals. Thanks.

Reply:
updated